minitests/partial_reconfig_flow/v2bit.sh

```diff
@@ -0,0 +1,39 @@
+set -ex
+
+fin=$1
+if [ -z "$fin" ] ; then
+    echo "need fin arg"
+    exit
+fi
+# fin=roi_blinky.v
+prefix=$(echo $fin |sed "s/\.v//")
+if [ "$fin" = "$prefix" ] ; then
+    echo "bad prefix"
+    exit
+fi
+echo "$fin => $prefix"
+
+make
+mkdir -p $prefix
+
+vivado -mode batch -source /dev/stdin <<EOF
+read_verilog $prefix.v
```

Comment:
FWIW, the Makefile has rules to go from .v to .dcp, .bit, and .fasm.  I think you can replace most of this with 'make ${prefix}_roi_partial.fasm', 'make ${prefix}.bit', and 'make ${prefix}_hand_crafted.bit'.  Those changes must have been submitted while you were working on this script.